### DISCUSSION TITLE
Use constant rather than a hard-coded string?

### DISCUSSION THREAD

**Discussion by rmccu** [jongpie/NebulaLogger#382] [Ideas]
https://github.com/jongpie/NebulaLogger/blob/3b04138cdb22211796870ef49d8b3773465641e9/nebula-logger/core/main/log-management/classes/LogEntryEventHandler.cls#L361

Seems like we should use a constant (private static final String) rather than a hard-coded string?

**Comment by jongpie** [comment]
Yeah, good suggestion, I prefer storing hardcoded strings as constants but overlooked this one. I'll get this updated in the next release.
